esp/esp/program/modules/handlers/programprintables.py

```diff
@@ -1835,6 +1835,57 @@ def time_possible(time, sections_list):
         response['Content-Disposition'] = 'attachment; filename=ok_times_concise.csv'
         return response
 
+    @aux_call
+    @needs_admin
+    def moderator_rooms_spr(self, request, tl, one, two, module, extra, prog):
```

Comment:
@willgearty I basically tried to combine code from the room schedules and the oktimes_spr, but honestly I was surprised that this code worked and am not 100% crystal clear on some of it so please read this code for me and make sure it's logical :) In particular, I don't know the difference between the event group and timeslots, so I'm not sure if there are cases where the code below wouldn't work.